HOWTO-RELEASE

```diff
@@ -1,24 +1,37 @@
-	Preparing a PROJ.4 Release
-	==========================
+Preparing a PROJ Release
+===============================================================================
 
-1) Update the version number in configure.ac (in AC_INIT()).
+Preparing a PROJ release is a three-step process. Before the actual release is
+published we want to issue one or more release candidates to ensure that the
+code is in fact ready to be released. That is step one. Step two starts when a
+release candidate has been selected for promotion to final release. Step three
+updates version numbers in the master branch to avoid confusion between
+released code and code still in development.
 
-   1a) Update the version number in PROJ_BUILD_VERSION in CMakeLists.txt
-   1b) Update the version number in doc/source/conf.py
-   1c) Update version numbers in man pages in man/man1/ and man/man3/
+The three phases are described below.
 
-2) Update the version number in proj_api.h (#define PJ_VERSION).
+1 Release candidate
+===============================================================================
 
-   2a) Update PROJ_VERSION_MAJOR, PROJ_VERSION_MINOR and PROJ_VERSION_PATCH
-       in src/proj.h
+At least one release candidate is necessary before the final release is
+published. In case more than one release candidate is needed most steps below
+can be skipped depending on what prompted the release candidate 1+n. Steps 1.6
+and 1.7 are mandatory for all release candidates, otherwise it will in most
+cases only be necessary to update the NEWS section (1.3).
 
-3) Update the version number, and date in src/pj_release.c.
+Release candidates should be released about a week before the final release is
+scheduled. When preparing for big releases it may be wise to issue the release
```

Comment:
Probably, yes. I was really just trying to say that if there's a huge amount of changes it may be smart to plan for some more time. That could also happen for a minor version release but is less likely. I'll try to reword the text.